.harness/dummytestpipeline.yaml

```diff
@@ -0,0 +1,83 @@
+pipeline:
+  name: dummy-test-pipeline
```

Comment:
Is this really a dummy pipeline? It looks useful and different from the simple lint pipeline. Can we just simplify for now and combine the pipeline? I.e. no need to run tests if lint is failing.